ui/css/treeherder-info-panel.css

```diff
@@ -103,15 +103,22 @@ div#info-panel .info-panel-navbar > ul.tab-headers > li {
   color: #9FA3A5;
 }
 
-.info-panel-navbar .navbar-nav > li > a:hover,
-.info-panel-navbar .navbar-nav > li > a:focus,
-.info-panel-navbar .navbar-nav > li > button:hover,
-.info-panel-navbar .navbar-nav > li > button:focus
+div#info-panel .navbar-nav > li > a:hover,
+div#info-panel .navbar-nav > li > a:focus,
+div#info-panel .navbar-nav > li > button:hover,
+div#info-panel .navbar-nav > li > button:focus
 {
   background-color: #1E252B;
   color: #D3D8DA;
 }
 
+div#info-panel .navbar-nav > li > a:active,
+div#info-panel .navbar-nav > li > button:active
+{
+  background-color: #000;
```

Comment:
Maybe this sounds crazy instead, but what about a 1s single rotation css animation on the `fa-repeat` icon? There would be no ambiguity or issues for folks less than keen eyesight. It might look nice and is purposeful too.

You could ease-in/out or whatever makes sense.